specs/altair/p2p-interface.md

```diff
@@ -74,6 +74,15 @@ New topics are added in Altair to support the sync committees and the beacon blo
 
 The specification around the creation, validation, and dissemination of messages has not changed from the Phase 0 document.
 
+The derivation of the `message-id` has changed starting with Altair to incorporate the message topic along with the message data.
```

Comment:
is this clearer?

```suggestion
The derivation of the `message-id` has changed starting with Altair to incorporate the message topic along with the message data.
This topic is found in the `topic` field of a gossipsub message.
```

Reply:
not entirely necessary but fine if you think it is clarifying

Reply:
Made an alternative suggestion in this spirit, that also clarifies the container type and missing-field case of protobuf.